app/models/user.rb

```diff
@@ -38,10 +38,15 @@ class User < ApplicationRecord
   aasm column: 'state' do
     state :new, initial: true
     state :transitioned
+    state :rolled_over
```

Comment:
Is it premature to use `rolled_over_2020` here?

Reply:
Next year we can always do `rolled_over_2021` if we need to. Not sure it's needed yet.